pkg/replica/replica.go

```diff
@@ -1136,6 +1136,10 @@ func (r *Replica) SetUnmapMarkDiskChainRemoved(enabled bool) {
 }
 
 func (r *Replica) Expand(size int64) (err error) {
+	if size%diskutil.VolumeSectorSize != 0 {
+		return fmt.Errorf("replica expanding size %v is not multiple for volume sector size %v", size, diskutil.VolumeSectorSize)
```

Comment:
`failed to expend volume replica size %v, because it is not multiple of volume sector size %v"`